dmd_Jmm.m

```diff
@@ -0,0 +1,26 @@
+function Jmm=dmd_Jmm(m,x,b)
```

Comment:
Also this function looks pretty much like it could be implemented in one line using the sum() command and and arrayfun or a bsxfun. This would make the file superfluous, things could be kept in distanceDiracGaussian()... See upper comment.